### PR TITLE
add installer url as return on imageSet

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # ([issue](https://issues.redhat.com/browse/THEEDGE-1373))

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: cmd/spec/main.go [hunk 1]
```diff
@@ -44,6 +44,12 @@ func main() {
 	}
 	components.Schemas["v1.ImageSetImagePackages"] = ImageSetImagePackages
 
+	ImageSetIntallerURL, _, err := openapi3gen.NewSchemaRefForValue(&routes.ImageSetIntallerURL{})
```
Comment (biancarosa) [inline cmd/spec/main.go:47]:
```suggestion
	ImageSetInstallerURL, _, err := openapi3gen.NewSchemaRefForValue(&routes.ImageSetInstallerURL{})
```

Comment (biancarosa) [reply cmd/spec/main.go:47]: not gonna suggest them all, but there are a bunch of these typos here :)

Comment (Avielyo10) [reply cmd/spec/main.go:47]: @biancarosa what a catch !

Comment (acosferreira) [reply cmd/spec/main.go:47]: Thanks for the catch @biancarosa

**Review by Avielyo10 — APPROVED** [review]

**Pull request merged**